omplbindings/scenes.go

```diff
@@ -115,34 +116,39 @@ func scene3() *config {
 		r3.Vector{X: 0., Y: 150., Z: 0.},
 		&spatialmath.R4AA{Theta: 0, RX: 0., RY: 0., RZ: 1.},
 	)
-	return &config{
-		Start:      startInput,
-		Goal:       spatialmath.NewPoseFromOrientation(goalPt, startPose.Orientation()),
-		RobotFrame: model,
-		WorldState: &commonpb.WorldState{
-			Obstacles: []*commonpb.GeometriesInFrame{
-				{
-					ReferenceFrame: "world",
-					Geometries: []*commonpb.Geometry{
-						{
-							Center: spatialmath.PoseToProtobuf(testPose),
-							GeometryType: &commonpb.Geometry_Box{
-								Box: &commonpb.RectangularPrism{DimsMm: &commonpb.Vector3{
-									X: 2000,
-									Y: 20,
-									Z: 120,
-								}},
-							},
+	worldState, err := referenceframe.WorldStateFromProtobuf(&commonpb.WorldState{
+		Obstacles: []*commonpb.GeometriesInFrame{
+			{
+				ReferenceFrame: "world",
+				Geometries: []*commonpb.Geometry{
+					{
+						Center: spatialmath.PoseToProtobuf(testPose),
+						GeometryType: &commonpb.Geometry_Box{
```

Comment:
You know, we initially structured these scenes to work on the proto structs directly...and they work, but what do you think about using the `NewBox` (or other appropriate shapes) here?

Reply:
Yeah I want to do this eventually, was just trying to get the MVP first